### PR TITLE
Authorize vpc to rds sg

### PR DESCRIPTION
**Pull request by ehowe**

### CODE REVIEW

**Comment by geemus** [comment]
Seems fine to me. @lanej could you take a look also? Thanks!

File: lib/fog/aws/requests/rds/delete_db_subnet_group.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@ def delete_db_subnet_group(name)
             raise Fog::AWS::RDS::NotFound.new("DBSubnetGroupNotFound => The subnet group '#{name}' doesn't exists")
           end
 
+          self.data[:subnet_groups].delete(name)
```
Comment (lanej) [inline lib/fog/aws/requests/rds/delete_db_subnet_group.rb]: Oops

**Comment by lanej** [comment]
Thanks @ehowe

**Comment by geemus** [comment]
Thanks!

**Pull request merged**